tests/common/reboot.py

```diff
@@ -33,7 +33,7 @@
     REBOOT_TYPE_POWEROFF: {
         "timeout": 300,
         "wait": 120,
-        "cause": "Power Loss",
+        "cause": "Power",
```

Comment:
Would this change impact other BRCM platforms such as Arista or Celestica?  Have you tried it out on those platforms?  Also not sure if this testcase is also ran against MLNX DUTs as well?  If so, will need to try out MLNX DUT to ensure it is not impacted...